src/components/InputContainer/index.tsx

```diff
@@ -33,6 +33,7 @@ export interface Props {
     uiMode?: UiMode;
     filled?: boolean;
     children?: React.ReactNode;
+    variant: 'form' | 'general';
```

Comment:
Shouldn't this be optional if there is a default value?

Reply:
Good point